### PR TITLE
FEATURE: page tree click actions

### PR DESCRIPTION
**Pull request by mgoldbeck**
Page tree click actions

- [x] onFocused
- [x] handle DoubleClick
- [x] status isActive Node
- [x] jump to page after click

### CODE REVIEW

**Comment by mgoldbeck** [comment]
PR is rebased and concentrating on the click actions as promised

File: Resources/Private/JavaScript/Host/Containers/LeftSideBar/PageTree/Node/index.js [hunk 1]
```diff
@@ -21,7 +26,11 @@ export default class Node extends Component {
             uri: PropTypes.string.isRequired,
             children: PropTypes.arrayOf(
                 PropTypes.string
-            )
+            ),
+            isDblClick: PropTypes.bool.isRequired
```
Comment (dimaip) [inline Resources/Private/JavaScript/Host/Containers/LeftSideBar/PageTree/Node/index.js]: What is this prop for?

File: Resources/Private/JavaScript/Host/Containers/LeftSideBar/PageTree/Node/index.js [hunk 2]
```diff
@@ -86,16 +92,31 @@ export default class Node extends Component {
...
 
     handleNodeLabelClick() {
-        const {item, onNodeClick, onNodeFocus} = this.props;
+        const {item, onNodeFocus, onNodeClick} = this.props;
 
         onNodeFocus(item.contextPath);
-        onNodeClick(item.uri);
+        onNodeClick(item.uri, item.contextPath);
+
+        if (this.props.item.isFocused) {
```
Comment (dimaip) [inline Resources/Private/JavaScript/Host/Containers/LeftSideBar/PageTree/Node/index.js]: Why do you put all of these `console.info` in here? If it's just for debugging, then let's not commit it.

Comment (mgoldbeck) [inline Resources/Private/JavaScript/Host/Containers/LeftSideBar/PageTree/Node/index.js]: Thought of that, but wanted to visualize the different type of clicks.

**Comment by grebaldi** [comment]
I'll merge this, so it won't break due to #359

**Pull request merged**